library/src/containers/AsyncApi/AsyncApi.tsx

```diff
@@ -98,6 +121,8 @@ class AsyncApiComponent extends Component<AsyncApiProps, AsyncApiState> {
 
     if (!(validatedSchema && validated)) return null;
 
+    console.log(validatedSchema);
```

Comment:
Was it forgotten here?

Reply:
Hah :D I always forget about console.log in my code. Thanks!